index.html

```diff
@@ -233,6 +233,6 @@ <h2 class="success__title">Изображение успешно загруже
   </template>
    <!--Подключил скрипт functiouns-->
    <script src= 'js/functions.js'></script>
-   <script src = 'js/main.js'></script>
+   <script src = 'js/main.js' type = 'module'></script>
```

Comment:
пробелы не ставятся у аттрибутов 
```suggestion
   <script src='js/main.js' type='module'></script>
```